swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -42,6 +42,9 @@ public final class Store: ObservableObject {
   private var vpnConfigurationManager: VPNConfigurationManager?
```

Comment:
If we make the manager/session a singleton instance, then we wouldn't need a cached IPCClient in Store.